content/en/continuous_integration/guides/developer_workflows.md

```diff
@@ -0,0 +1,84 @@
+further_reading:
+  - link: "https://www.datadoghq.com/blog/datadog-github-actions-ci-visibility/"
+    tag: "Blog"
+    text: "Monitor your GitHub Actions workflows with Datadog CI Visibility"
+  - link: "/integrations/guide/source-code-integration"
+    tag: "Documentation"
+    text: "Learn about the GitHub integration"
+---
+
+## Overview
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+CI Test Visibility integrates with other developer-oriented Datadog products as well as external partners such as GitHub to streamline developer workflows. Relevant features include:
+- Create and open GitHub issues 
+- Open tests in GitHub and your IDE
+- [Enabling Test Summaries in GitHub Pull Request Comments][1]
+
+## Create and open GitHub issues
+From Test Visibility, you can create and open pre-filled GitHub issues with relevant context into your tests as well as deep links back to Datadog for more streamlined debugging workflows. Creating issues directly from Test Visibility can help you track and maintain accountability for test failures and flaky tests.
+
+### In-app entry points
+
+You can create pre-filled GitHub issues from three areas within CI Test Visibility:
+1. Commit Overview (from Commits table) 
+2. Branch Overview 
+3. Test Detail View
+
+#### Commit Overview
+{{< img src="ci/github_issues_light.png" alt="Datadog GitHub issues preview" style="width:100%;">}}
+
+The overview page for any commit can be discovered via a particular branch or from within any particular test. 
+
+From the Commit Overview page, click on any row in the `Failed Tests` or `New Flaky Tests` tables and select `Open issue in GitHub`. 
+
+#### Branch Overview
+From this page, click on any row in the `Flaky Tests table` and select `Open issue in GitHub`.
+{{< img src="ci/github_issues_flaky_light.png" alt="Datadog GitHub issues flaky tests table preview" style="width:100%;">}}
+
+#### Test Detail View
+From within a specific test run, click the `Actions` button at the top right and select `Open issue in GitHub`. 
+{{< img src="ci/github_issues_detail_light.png" alt="Datadog GitHub issues test detail view preview" style="width:100%;">}}
+
+You also have the option to copy an issue description in Markdown for pasting test details elsewhere. The Markdown description will contain information such as the test execution link, service, branch, commit, author, and error. 
+{{< img src="ci/github_issues_markdown.png" alt="Copy issue description in Markdown format for GitHub issues" style="width:30%;">}}
+
+### Sample GitHub issue
+Below is what a pre-filled GitHub issue might look like:
+{{< img src="ci/prefilled_github_issue.png" alt="Pre-filled GitHub issue" style="width:50%;">}}
+
+### Compatibility
+This is available for all Test Visibility customers. It does not require usage of the GitHub App integration.
```

Comment:
Can you please clarify what "This" is referring to, exactly?

Reply:
"Creating and opening GitHub issues" - will change this to be more specific!